packages/contracts-core/contracts/manager/LightManager.sol

```diff
@@ -53,6 +55,13 @@ contract LightManager is AgentManager, InterfaceLightManager {
         __Ownable_init();
     }
 
+    // ════════════════════════════════════════════════ OWNER ONLY ═════════════════════════════════════════════════════
+
+    /// @inheritdoc InterfaceLightManager
+    function setAgentRootWhenStuck(bytes32 agentRoot_) external onlyOwner onlyWhenStuck {
```

Comment:
ToB-05 suggests a two step function with a time delay, so consider adding a second step similar to OZ's [`OwnableTwoStep`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable2Step.sol).

Reply:
Could go for two-step process here, agreed